zproject/dev_settings.py

```diff
@@ -22,5 +22,7 @@
 CAMO_URI = ''
 OPEN_REALM_CREATION = True
 TERMS_OF_SERVICE = 'zproject/terms.md.template'
+# Leave as '0.0' if TERMS_OF_SERVICE is None
+TOS_VERSION = '0.0'
```

Comment:
I don't think we need to set this in `dev_settings.py` -- it gets the default from `zproject/settings.py` anyway